### PR TITLE
📝 Performance doc

### PR DESCRIPTION
**Pull request by RTLS**

### CODE REVIEW

**Comment by RTLS** [comment]
* **#736** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/736?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/736?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: docs/performance.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ id              | integer                     |           | not null | nextval('benchmark_records_id_seq'::regclass)
+ string_field    | text                        |           |          |
+ numeric_field   | numeric                     |           |          |
+ timestamp_field | timestamp with time zone    |           |          |
+ json_field      | jsonb                       |           |          |
+ inserted_at     | timestamp without time zone |           |          | now()
+ updated_at      | timestamp without time zone |           |          | now()
+```
+
+Records delivered to Kafka are annotated with a `delivered_at` timestamp. Full end-to-end latency is calculated by subtracting the `delivered_at` timestamp from the `updated_at` timestamp.
```
Comment (acco) [inline docs/performance.mdx:57]: I assume `updated_at` is set by the script on updates?

File: docs/performance.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+| Tool | Latency at 2k ops/s |
+|------|-------------------|
+| **Sequin** | **192ms** |
+| Debezium | 234ms |
+| PeerDB | 30 seconds |
+| Fivetran | 5 minutes |
+| Airbyte | 1+ hours |
+
+## Test Setup
```
Comment (acco) [inline docs/performance.mdx:18]:
```suggestion
## Test setup
```

File: docs/performance.mdx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+ numeric_field   | numeric                     |           |          |
+ timestamp_field | timestamp with time zone    |           |          |
+ json_field      | jsonb                       |           |          |
+ inserted_at     | timestamp without time zone |           |          | now()
+ updated_at      | timestamp without time zone |           |          | now()
+```
+
+Records delivered to Kafka are annotated with a `delivered_at` timestamp. Full end-to-end latency is calculated by subtracting the `delivered_at` timestamp from the `updated_at` timestamp.
+
+### Throughput Stability
```
Comment (acco) [inline docs/performance.mdx:59]:
```suggestion
### Throughput stability
```

File: docs/performance.mdx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+By comparison, Debezium achieves a slightly higher latency of 234ms at similar throughput:
+
+```bash
+Average throughput: 2000.6 ops/sec
+Average latency: 234ms
+99th percentile latency: 286ms
+```
+
+### Resource Utilization
```
Comment (acco) [inline docs/performance.mdx:77]:
```suggestion
### Resource utilization
```

File: docs/performance.mdx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+The Sequin ECS task makes efficient use of provisioned resources, employing parallelism with lightweight processes. During the test, the Sequin ECS task averaged 34% CPU utilization and 10% memory utilization.
```
Comment (acco) [inline docs/performance.mdx:79]:
```suggestion
Sequin was deployed to EC2 via Elastic Container Service (ECS). During the test, the Sequin ECS task averaged 34% CPU utilization and 10% memory utilization.
```

File: docs/performance.mdx [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+## Next Steps
+
+Ready to see Sequin's performance for yourself?
+
+- [Quick Start Guide](/quickstart)
```
Comment (acco) [inline docs/performance.mdx:85]:
```suggestion
- Check out our [quickstart with Kafka](/quickstart/kafka)
```

File: docs/performance.mdx [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+- [Deploy to Production](/how-to/deploy-to-production)
```
Comment (acco) [inline docs/performance.mdx:86]:
```suggestion
- See our how-to on [deploying to production](/how-to/deploy-to-production)
```

File: docs/performance.mdx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+- [Compare to Other CDC Tools](/compare)
```
Comment (acco) [inline docs/performance.mdx:87]:
```suggestion
- [Compare Sequin to other CDC tools](/compare)
```

File: docs/performance.mdx [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+99th percentile latency: 286ms
+```
+
+### Resource Utilization
+
+The Sequin ECS task makes efficient use of provisioned resources, employing parallelism with lightweight processes. During the test, the Sequin ECS task averaged 34% CPU utilization and 10% memory utilization.
+
+## Next Steps
+
+Ready to see Sequin's performance for yourself?
```
Comment (acco) [inline docs/performance.mdx:83]: You could maybe even use this component:

```
<CardGroup cols={2}>
  <Card title="Guide: Connect Postgres" icon="elephant" href="/connect-postgres">
    Connect your Postgres database to Sequin.
  </Card>
  <Card title="Guide: Setting up an SQS sink" icon="aws" href="/how-to/stream-postgres-to-sqs">
    Setup an SQS sink to send changes to your queue.
  </Card>
</CardGroup>
```

File: docs/performance.mdx (hunk 1 above)
Comment (RTLS) [reply docs/performance.mdx:57]: correct

**Pull request merged**